bioblend/galaxy/workflows/__init__.py

```diff
@@ -555,7 +567,7 @@ def invoke_workflow(self, workflow_id, inputs=None, params=None,
           (which is stable across workflow imports) or the step UUID which is
           also stable.
         """
-        payload = {'workflow_id': workflow_id}
+        payload: Dict[str, Union[str, bool, dict]] = {'workflow_id': workflow_id}
```

Comment:
```suggestion
        payload = {'workflow_id': workflow_id}
```

Reply:
Without a type hint here linting fails with the following errors:

```
GLOB sdist-make: /home/fread/uni/ws2021/master-project/bioblend/setup.py
lint inst-nodeps: /home/fread/uni/ws2021/master-project/bioblend/.tox/.tmp/package/1/bioblend-0.15.0.zip
lint installed: attrs==20.3.0,bioblend @ file:///home/fread/uni/ws2021/master-project/bioblend/.tox/.tmp/package/1/bioblend-0.15.0.zip,boto==2.49.0,certifi==2020.12.5,chardet==4.0.0,flake8==3.9.0,flake8-bugbear==21.4.3,flake8-import-order==0.18.1,idna==2.10,mccabe==0.6.1,mypy==0.812,mypy-extensions==0.4.3,pycodestyle==2.7.0,pyflakes==2.3.1,PyYAML==5.4.1,requests==2.25.1,requests-toolbelt==0.9.1,typed-ast==1.4.2,typing-extensions==3.7.4.3,urllib3==1.26.4
lint run-test-pre: PYTHONHASHSEED='1274784429'
lint run-test: commands[0] | flake8 .
lint run-test: commands[1] | mypy bioblend/
bioblend/galaxy/workflows/__init__.py:570:33: error: Incompatible types in assignment (expression has type "Dict[Any, Any]", target has type "str")
bioblend/galaxy/workflows/__init__.py:573:37: error: Incompatible types in assignment (expression has type "Dict[Any, Any]", target has type "str")
bioblend/galaxy/workflows/__init__.py:576:45: error: Incompatible types in assignment (expression has type "Dict[Any, Any]", target has type "str")
bioblend/galaxy/workflows/__init__.py:583:44: error: Incompatible types in assignment (expression has type "bool", target has type "str")
bioblend/galaxy/workflows/__init__.py:585:55: error: Incompatible types in assignment (expression has type "bool", target has type "str")
bioblend/galaxy/workflows/__init__.py:589:48: error: Incompatible types in assignment (expression has type "bool", target has type "str")
Found 6 errors in 1 file (checked 65 source files)
```

Reply:
Just `payload: dict =`  also works.

Reply:
Thanks, this stuff is also new for me! I've pushed a commit to use the `Dict[str, Any]` annotation, which is kind of a middle ground.